tests/infra_test.go

```diff
@@ -1141,6 +1143,9 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 
 		Context("when the controller pod is not running and an election happens", func() {
 			It("[test_id:4642]should succeed afterwards", func() {
+				// This test needs at least 2 controller pods. Skip on single-replica.
```

Comment:
Do you think it would make sense to have an alternative test for SingleReplica?

Reply:
I can't think of such a test, as you can't elect a leader in a population of 0! But maybe you have something else in mind?

Reply:
I thought about ensuring the pod will be re-created and it gets elected. But I am not sure if it has value...